### ISSUE TITLE
[feature request] --share flag?

### ISSUE THREAD

**Issue by yesbroc** [comfyanonymous/ComfyUI#996]
sometimes i cant use my pc so i have to use sharelink if i want to play around with models, if this already exists can somebody tell me how to get it?

**Comment by dotgrid** [comment]
> sometimes i cant use my pc so i have to use sharelink if i want to play around with models, if this already exists can somebody tell me how to get it?

Yes, you can set the host to 0.0.0.0, which will allow access from other devices. Additionally, you'll need to open any required ports on your firewall, and so on.

**Comment by yesbroc** [comment]
does this work if im on a separate network?

**Comment by marclundgren** [comment]
> Yes, you can set the host to 0.0.0.0, which will allow access from other devices. Additionally, you'll need to open any required ports on your firewall, and so on.

How do we set the host to `0.0.0.0`?
